doc/integrator/upgrade_application.rst

```diff
@@ -406,6 +406,36 @@ Once restored, set the original names back:
       ALTER TABLE <schema_name>.layer ADD CONSTRAINT <schema_name>.layer_id_fkey FOREIGN KEY (id) REFERENCES <schema_name>.treeitem(id);
       ALTER TABLE <schema_name>.layer DROP CONSTRAINT <schema_name>.layertmp_id_fkey;
 
+Edition
++++++++
+
+If you migrate editable tables, you need to modify the geometry data types to match Postgis 2 new Typmod.
+
+Example for a layer with ``Point`` geometries and a 21781 projection:
+
+   .. code:: sql
+
+      ALTER TABLE my_table ALTER COLUMN geom SET DATA TYPE geometry(Point, 21781);
+
+To help doing it on several tables at once, here is a function and an example of usage:
+
+   .. code:: sql
+
+      CREATE OR REPLACE FUNCTION migrategeomtopostgis2(_tablename text, _geomcolumn text, _geomtype text, _srid int) RETURNS void AS $$
+      DECLARE
+      _cleangeomtype text;
+      BEGIN
+      _cleangeomtype := initcap(lower(_geomtype));
+      EXECUTE ' ALTER TABLE ' || _tablename || ' ALTER COLUMN ' || _geomcolumn || ' SET DATA TYPE geometry(' || _cleangeomtype || ', ' || _srid || ')';
```

Comment:
afaik the method doesnt accept dimension as parameter, the dimensions constraint are automatically set based on the geometry type
and I assume the geometry type is correct, otherwise it would be too complicated (and Im not going to support weird cases)